collector.go

```diff
@@ -59,6 +59,24 @@ var (
 		"Atmospheric pressure measurement in mb",
 		varLabels,
 		nil)
+
+  windDesc = prometheus.NewDesc(
+    prefix+"wind_kmh",
```

Comment:
Usually one should use base SI units for metrics, but in this case it might be more convenient to just use kilometers per hour. Just a suggestion: The English abbreviation is "kph" and I would also state that it is the wind strength in the name, so maybe: `wind_strength_kph`?